Filter/ModelFilter.php

```diff
@@ -24,19 +24,19 @@ class ModelFilter extends Filter
      */
     public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
     {
-        if (!$data || !is_array($data) || !array_key_exists('value', $data)) {
+        if (!$data || !is_array($data) || !array_key_exists('value', $data) || empty($data['value'])) {
```

Comment:
@samuel4x4 I have an issue with this : now I can't filter by `null`. I want to see all entries that miss a given association, how do I do that?